sha1/README.md

```diff
@@ -9,7 +9,7 @@
 
 Pure Rust implementation of the [SHA-1] cryptographic hash algorithm.
 
-## 🚨 Warning: Cryptographically Broken! 🚨
+## Warning: Cryptographically Broken!
```

Comment:
The emojis are removed because they cause ICE on the latest Nightly. We can return them later after the bug is fixed.